src/java.desktop/macosx/classes/sun/lwawt/macosx/CPrinterJob.java

```diff
@@ -46,6 +46,7 @@
 import javax.print.attribute.standard.MediaSizeName;
 import javax.print.attribute.standard.PageRanges;
 import javax.print.attribute.standard.Sides;
+import javax.print.attribute.standard.OutputBin;
```

Comment:
Please sort the import...Guess OutputBin should come before PageRanges

Reply:
Updated.